AirCasting/CreateSessionViews/SelectPeripheralView.swift

```diff
@@ -95,7 +96,7 @@ struct SelectPeripheralView: View {
     
     var titleLabel: some View {
         var title: Text
-        if syncMode == true {
+        if syncMode == true || SDClearingRouteProcess {
             title = Text(Strings.SelectPeripheralView.titleSyncLabel)
```

Comment:
Should this be the same for clearing SD card? "Select the device you'd like to sync"?

Reply:
Great catch, I will make a change then